packages/datadog-plugin-apollo-gateway/src/index.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const CompositePlugin = require('../../dd-trace/src/plugins/composite')
+const ApolloGatewayExecutePlugin = require('./execute')
+const ApolloGatewayPostProcessingPlugin = require('./postprocessing')
+const ApolloGatewayRequestPlugin = require('./request')
+const ApolloGatewayPlanPlugin = require('./plan')
+const ApolloGatewayValidatePlugin = require('./validate')
+const ApolloGatewayFetchPlugin = require('./fetch')
+const ApolloGatewayGeneralPlugin = require('./general')
+
+class ApolloGatewayPlugin extends CompositePlugin {
+  static get id () { return 'apollo-gateway' }
+  static get plugins () {
+    return {
```

Comment:
I'm not 100% about the usage of `CompositePlugin` here. I think it would make more sense only if Apollo was at the root with a `gateway` plugin on the composite, because that is also exposed to users a way to disable individual composed plugins.